### PR TITLE
Fix nav-spot, to display placeholder box when no media. DDFFORM-309

### PR DESCRIPTION
**Pull request by rasben**
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/779

### CODE REVIEW

**Review by kasperg — APPROVED** [review]
:+1: LGTM

File: src/stories/Library/nav-spot/NavSpot.tsx [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
       <a href="#" className="nav-spot__content">
-        {media ? <figure className="nav-spot__media">{media}</figure> : ""}
+        <figure className="nav-spot__media">
+          {media || (
+            <div className="nav-spot__placeholder-text">{placeholderText}</div>
+          )}
+        </figure>
+
         <div className="nav-spot__text">
           <h1 className="nav-spot__title">{title}</h1>
```
Comment (kasperg) [inline src/stories/Library/nav-spot/NavSpot.tsx:35]: It is outside the scope of this PR and heading levels are tricky but I would never expect a `<h1>` to be the right heading level for a Nav Spot.

**Review by kasperg — APPROVED** [review]
I had a few thoughts the second time around :smile:.

File: src/stories/Library/nav-spot/NavSpot.tsx (hunk 1 above)
Comment (rasben) [reply src/stories/Library/nav-spot/NavSpot.tsx:35]: It's wrapped in a `<article>`, so yeah, h1 is appropiate :) @kasperg

File: src/stories/Library/nav-spot/NavSpot.tsx [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
+const NavSpot: FC<NavSpotProps> = ({
+  variant,
+  title,
+  subtitle,
+  media,
+  placeholderText,
+}) => {
+  const classes = clsx("nav-spot", "arrow__hover--right-large", {
+    "nav-spot--has-no-media": !media,
+    "nav-spot--has-media": media,
```
[the remark this answers is not in the document]
Comment (rasben) [reply src/stories/Library/nav-spot/NavSpot.tsx:22]: Good point - I dont use "has-media" - it is something that is used in cards.

**Pull request merged**